src/common/marlin_vars.hpp

```diff
@@ -176,24 +179,23 @@ typedef enum {
 
 // variables structure - used in server and client
 // deliberately ordered from longest data types to shortest to avoid alignment issues
-typedef struct _marlin_vars_t {
+struct marlin_vars_t {
```

Comment:
Again, the naming…

Reply:
I wanted to rename this one in next PR, we already have too many changes in this one.